src/website/views/templates/authentication.pug

```diff
@@ -27,21 +27,29 @@ h3 App Basic Sign In
             a.a-btn.-flat.-icon.-light(href="#")
               .a-btn__content
                 i.a-icon.icon-circle-question
-      .chi-app__body.-px-sm--2.-py-sm--4.-d--flex.-justify-content--center
-        .-pb--2.-w--100.-mw--480
+      .chi-app__body.-px-sm--2.-py-sm--4
+        .-w--100.-m--auto.-mw--480
           .a-card.-b--0.-bg--none
-            .a-card__content.-text--center.-px--4.-px-sm--6.-pb--4
-              h3.-text--normal Sign In
-              .a-inputWrapper.-floatingLabel.-pt--2#aemail
-                input(type='email', class='a-input -xl', id='email')
-                label(for='email') Email
-              .a-inputWrapper.-floatingLabel.-pt--1#apassword
-                input(type='password', class='a-input -xl', id='password')
-                label(for='password') Password
-              .m-formGroup.-pt--3.-pt-sm--4.-justify-content--between.-align-items--center.-flex--column.-flex-sm--row
-                a.-text.-mb--2.-mb-sm--0(href="#") Forgot email or password?
-                button.a-btn.-primary.-xl Sign In
-          .-text.-text--center.-py--2 Don't have an account? <a href="#">Sign Up</a>
+            .a-card__content.-px--4.-px-sm--6
+              fieldset
+                .a-grid.-noGutter.-justify-content--center.-align-items--center
+                  div.a-col.-w--12
+                    h3.-text--normal.-text--center Sign In
+                  div.a-col.-w--12
+                    .a-inputWrapper.-floatingLabel.-pt--2#aemail
+                      input(type='email', class='a-input -xl', id='email')
+                      label(for='email') Email
+                  div.a-col.-w--12.-mb--3.-mb-sm--4
```

Comment:
@davidojedacl @mattnickles just a doubt on these margins .-mb-sm--4 should be .-mb-md--(n) or should we removed and just let .-mb--3? Thoughts?